convert2rhel/utils.py

```diff
@@ -144,9 +144,10 @@ def run_subprocess(cmd, print_cmd=True, print_output=True):
     )
     output = ""
     for line in iter(process.stdout.readline, b""):
-        output += line.decode()
+        line = line.decode("utf8")
+        output += line
```

Comment:
Unit tests are failing here

Reply:
I have taken care of this error message in the unit test and will push the changes!